hnn_core/tests/test_extracellular.py

```diff
@@ -282,3 +282,7 @@ def test_extracellular_viz():
 
     with pytest.deprecated_call():
         net.rec_arrays['arr1'].plot_lfp(show=False, tmin=10, tmax=100)
+    with pytest.raises(RuntimeError, match='Please use sink = '"'b'"' or '
```

Comment:
```suggestion
    with pytest.raises(RuntimeError, match='Please use sink = "b" or '
```

python trick ... you just need to use double quotes inside single quotes or single quotes inside double quotes and it will work

Reply:
ah, good to know I was breaking my ears on this one :)